admission-control/admission-control-service/src/admission_control_service.rs

```diff
@@ -71,17 +68,20 @@ impl AdmissionControlService {
         );
         runtime
     }
-
     /// Pass the UpdateToLatestLedgerRequest to Storage for read query.
-    async fn update_to_latest_ledger_inner(
+    fn update_to_latest_ledger_inner(
         &self,
         req: UpdateToLatestLedgerRequest,
     ) -> Result<UpdateToLatestLedgerResponse> {
         let rust_req = libra_types::get_with_proof::UpdateToLatestLedgerRequest::try_from(req)?;
-        let (response_items, ledger_info_with_sigs, epoch_change_proof, ledger_consistency_proof) =
-            self.storage_read_client
-                .update_to_latest_ledger(rust_req.client_known_version, rust_req.requested_items)
-                .await?;
+        let (
+            response_items,
+            ledger_info_with_sigs,
+            epoch_change_proof,
+            ledger_consistency_proof,
+        ) = self
+            .db
+            .update_to_latest_ledger(rust_req.client_known_version, rust_req.requested_items)?;
```

Comment:
someone just said this was deprecated!

Reply:
AC is going to be deprecated and I don't think it is worth investigating how to refactor AC now.